guide/popular-topics/common-questions.md

```diff
@@ -59,6 +59,28 @@ if (<message>.author.id === '<id>') {
 }
 ```
 
+### How do I check which role was added/removed, and for which member?
+
+<!-- eslint-skip -->
+
+```js
+// We start by declaring a guildMemberUpdate listener
+// This should ***not*** be inside the message listener (or any other listener)
```

Comment:
I don't think this reads very well, I'd suggest something along the lines of:
```suggestion
// This code should be placed outside of any other listener callbacks to prevent listener nesting
```